build.gradle

```diff
@@ -183,5 +183,16 @@ jreleaser {
   }
 }
 
+// Remove unwanted "scan" directory after publishing
```

Comment:
If this is done "after publishing" would that mean the additional artifact would have been already uploaded to Central?